tests/data/BatcacheVariantTest.inc

```diff
@@ -0,0 +1,12 @@
+<?php
+
+die(); //Don't actually run the following code.
```

Comment:
We can safely omit the die()...this only gets read in as text, not included/required as PHP so it won't be executed.